plugins/destination/bigquery/client/deletestale.go

```diff
@@ -2,13 +2,12 @@ package client
 
 import (
 	"context"
-	"errors"
-	"time"
 
-	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v4/message"
+	"github.com/cloudquery/plugin-sdk/v4/plugin"
 )
 
 // DeleteStale is not currently implemented for BigQuery, as it only supports "append" write mode.
-func (*Client) DeleteStale(ctx context.Context, tables schema.Tables, sourceName string, syncTime time.Time) error {
-	return errors.New("DeleteStale not implemented")
+func (*Client) DeleteStale(context.Context, message.WriteDeleteStales) error {
```

Comment:
could just embed `batchwriter.UnimplementedDeleteStale` to `Client` and remove this file

Reply:
updated